echion/strings.h

```diff
@@ -70,6 +81,42 @@ pyunicode_to_utf8(PyObject *str_addr)
     return dest;
 }
 
+
+// ----------------------------------------------------------------------------
+#if PY_VERSION_HEX >= 0x030c0000
+
+static long long
+pylong_to_llong(PyObject *long_addr)
```

Comment:
We should probably move this to a `long.h` source as this is not related to strings